ext/AvizExt/viz/spatial.jl

```diff
@@ -251,6 +256,75 @@ function ADRIA.viz.map!(
         axis_opts
     )
 end
+function ADRIA.viz.map(
```

Comment:
Sounds good :)

Reply:
I prefer the current approach but I'm not 100% up on the context here, just latching onto the suggestion to create a separate map plotting function. Julia is not strictly an object oriented language and isn't intended to be used like Python or Java.

The advantage of having multiple dispatch (or something like it, as in R) is to prevent everyone from having to remember a lot of function names. Is it `map` or `vizualize_selection_criteria_as_map` or `display_map` or `plot_selection_criteria_map` or...

As a user I don't want to have to remember endless possible function names for specific use cases. I just want a map.

Instead, this behaviour of:

1. You have a vector of results -> `map()` produces a single map.
2. You have a matrix of results -> `map()` produces a series of maps.

is sensible to me.

But again, I don't have full context.

https://www.juliaopt.org/meetings/santiago2019/slides/stefan_karpinski.pdf

Reply:
I understand and I agree with what you said. 

My problem with how this function is defined right now is that it doesn't plot selection criteria scores in particular. It plots multiple maps with whatever you pass as an argument. But still the names of variables and the plots default titles are related to this specific kind of plot (multiple maps with selection criteria scores). It's like having a function called `sum` and the arguments are called `prime_number_a` and `prime_number_b`. If it sums more than just prime numbers, they should be called `number_a` and `number_b` or whatever.

About the need for a specific function called `selection_criteria_map`, maybe we don't need one, I agree. depending on how this `map` function with multiple maps is defined we can just use this one and that's fine. My comment was more about making this `map` general than about the need of a specific function with a specific name for this kind of plot.

Reply:
Incidentally @Rosejoycrocker the variable names/titles displayed in the figure should be run through `human_readable()`